packages/gatsby-telemetry/src/postinstall.js

```diff
@@ -1,15 +1,13 @@
+const notice = require(`./notice`)
+
 let enabled = false
 try {
   const ci = require(`ci-info`)
   const Configstore = require(`configstore`)
   const config = new Configstore(`gatsby`, {}, { globalConfigPath: true })
   enabled = config.get(`telemetry.enabled`)
   if (enabled === undefined && !ci.isCI) {
-    console.log(
-      `Gatsby has started collecting anonymous usage analytics to help improve Gatsby for all users.\n` +
-        `If you'd like to opt-out, you can use \`gatsby telemetry --disable\`\n` +
-        `To learn more, checkout http://gatsby.dev/telemetry`
-    )
+    notice()
```

Comment:
I'm more in favour of renaming the function or putting the text back here and in telemetry.

if we rename it I would go for
```suggestion
    showAnalyticsNotification()
```

Reply:
Done 👌